components/brave_wallet/browser/keyring_service.h

```diff
@@ -167,7 +167,7 @@ class KeyringService : public KeyedService, public mojom::KeyringService {
       bool is_eip712 = false);
 
   std::vector<uint8_t> SignMessageBySolanaKeyring(
-      const mojom::AccountId& account_id,
+      const mojom::AccountIdPtr& account_id,
```

Comment:
`mojom::AccountIdPtr` since `mojo::StructPtr` is a move-only smart pointer.

Reply:
With `const mojom::AccountIdPtr&` it will use `mojo::InlinedStructPtr`

Reply:
nvm, let's stick with original way otherwise we have to change chain of callers to also move `mojo::AccountInfo`